src/main/webapp/WEB-INF/jsp/Main.jsp

```diff
@@ -10,13 +10,13 @@
   <title>ÍsFar</title>
   <link href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css' rel='stylesheet' type='text/css'>
   <link href='/resources/theme1/css/main.css' rel='stylesheet' type='text/css'>
-  <script src="/resources/theme1/build/jquery-1.11.3.min.js" /></script>
+  <script src="/resources/theme1/build/jquery-1.11.3.min.js"></script>
```

Comment:
wrong syntax,

but I am curious about this, what stuff would you put between the  script tags? 
e.g. `<script src="/resources/theme1/build/jquery-1.11.3.min.js" >WHAT WOULD YOU PUT HERE??</script>`